Dockerfile

```diff
@@ -17,6 +17,11 @@
 # Build the manager binary
 # Run this with docker build --build-arg builder_image=<golang:x.y.z>
 ARG builder_image
+
+# Ignore Hadolint rule "Always tag the version of an image explicitly."
+# It's an invalid finding since the image is explicitly set in the Makefile.
+# https://github.com/hadolint/hadolint/wiki/DL3006
+# hadolint ignore=DL3006
```

Comment:
Can this be ignored globally instead of having it in each Dockerfile?

Reply:
@vincepri Yes, it is possible to have global ignore rules. In an earlier review it was requested per file: https://github.com/kubernetes-sigs/cluster-api/pull/6886#discussion_r917834849 I have no strong opinion here so whatever the community thinks is best I can implement.

Reply:
I still think it's more explicit to do it where we need so we catch this if it's changed in e.g. a new Dockerfile.

Reply:
I can see that, that's completely fine, it was just noisy when reviewing the PR